### ISSUE TITLE
PnP Search Web Parts unavailable to add to pages unless installed to "all sites"

### ISSUE THREAD

**Issue by erobillard** [microsoft-search/pnp-modern-search#3856]
**Version used**
4.12.*

**Describe the bug**
On sites where the PnP Search web parts are installed as an App, they do not appear when adding web parts to the page. Existing PnP Search web parts work and can be edited, but new ones cannot be added with the Add Web Part UI. However, if the package is deployed to "all sites" then all the PnP Search web parts are available to add to any page on any site. 

I believe this is a side effect of Microsoft eliminating the SharePoint App Store as it seemed to coincide with that July 1 change for all tenants. This may end up as something the PnP project has no control over to remedy, which would mean a "fix" to the documentation rather than the code base.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Install PnP Modern Search to any tenant, do **not** select the option to deploy the app to all sites. If not prompted to accept requested Graph permissions, open the API approval page and accept all requested approvals related to the PnP Modern Search package. 
2. Open the settings icon on any SharePoint Site. Select Add an App. Select the PnP Modern Search app. 
3. Confirm that the app is installed by navigating to Site Contents. 
4. Open any Page in the Pages library. Edit the page. Add a web part to any section. Search for PnP. The results are an empty set.

**Expected behavior**
1. Return to step 1 and now provision the app to All sites. Re-approve permissions requested as needed. 
2. Step 4 (above) will now work as expected. 

**Desktop (please complete the following information):**
 - MS Edge Version 126.0.2592.87 (Official build) (64-bit)

**Additional context**
The fundamental issue is having to deploy the package to all sites, when typically they are only deployed to specific Search Centre sites. Unless this is somehow a configuration issue with a manifest file and not a Microsoft issue, the resolution will be to update the docs and not the package. 

Awesome work by the way - can never give this project's contributors anough kudos, it's a wonderful app and the functionality is now far beyond the Search Center it originally replaced. Much appreciated!

**Comment by wobba** [comment]
Is this specific to the PnP Modern Search web parts? Or a generic SPFx issue to be logged at https://github.com/SharePoint/sp-dev-docs?

**Comment by erobillard** [comment]
Hi Mikael, thanks for the suggestion. I've logged the issue there as well with a request for anyone to confirm or refute what I'm seeing. Since the web parts began showing up immediately after I redeployed to All sites I suspect this is new behaviour (perhaps related to the July 1 changes) though it's possible I've missed a configuration change somewhere outside the scope of app management. 

SPFx issue logged: 
https://github.com/SharePoint/sp-dev-docs/issues/9799

Have a great day,
-Eli.
